### PR TITLE
dribblish-dynamic.js: reduce redundancy

### PR DESCRIPTION
**Pull request by AvinashReddy3108**
Also fix the brightness threshold for light/dark color switching.

### CODE REVIEW

**Comment by morpheusthewhite** [comment]
@JulienMaille

File: DribbblishDynamic/dribbblish-dynamic.js [hunk 1]
```diff
@@ -188,12 +188,10 @@ function getAlbumInfo(uri) {
 }
 
 function isLight(hex) {
-    var bigint = parseInt(hex.replace("#",""), 16);
```
Comment (JulienMaille) [inline DribbblishDynamic/dribbblish-dynamic.js:191]: please convert strings to integers
`var [r,g,b] = hexToRgb(hex).split(',').map(Number);`

**Review by JulienMaille — CHANGES_REQUESTED** [review]

**Comment by JulienMaille** [comment]
Can you please add this too?
````js
function LightenDarkenColor(hex, amt) {
    var [r,g,b] = hexToRgb(hex).split(',').map(Number);
    r = Math.max(0, r + amt);
    b = Math.max(0, b + amt);
    g = Math.max(0, g + amt);
    var newColor = g | (b << 8) | (r << 16);
    return '#'+newColor.toString(16);
}
```

**Comment by AvinashReddy3108** [comment]
> Can you please add this too?
> 
> ```js
> function LightenDarkenColor(hex, amt) {
>     var [r,g,b] = hexToRgb(hex).split(',').map(Number);
>     r = Math.max(0, r + amt);
>     b = Math.max(0, b + amt);
>     g = Math.max(0, g + amt);
>     var newColor = g | (b << 8) | (r << 16);
>     return '#'+newColor.toString(16);
> }
> ```
> ```

https://github.com/AvinashReddy3108/spicetify-themes/commit/0f090cad009dd46598f8f2ca82d90edfcf8c25a5

File: DribbblishDynamic/dribbblish-dynamic.js (hunk 1 above)
Comment (AvinashReddy3108) [reply DribbblishDynamic/dribbblish-dynamic.js:191]: done

File: DribbblishDynamic/dribbblish-dynamic.js [hunk 2]
```diff
@@ -188,12 +188,10 @@ function getAlbumInfo(uri) {
...
-    var r = (bigint >> 16) & 255;
-    var g = (bigint >> 8) & 255;
-    var b = bigint & 255;
+    var rgb = hexToRgb(hex);
```
Comment (JulienMaille) [inline DribbblishDynamic/dribbblish-dynamic.js:191]: this line is useless now

**Review by JulienMaille — CHANGES_REQUESTED** [review]

**Review by JulienMaille — APPROVED** [review]

File: DribbblishDynamic/dribbblish-dynamic.js (hunk 2 above)
Comment (JulienMaille) [reply DribbblishDynamic/dribbblish-dynamic.js:191]: I'll do it

**Pull request merged**